public/index.html

```diff
@@ -5,13 +5,17 @@
     <title>Mashbnb - proxy</title>
   </head>
   <body>
-    <div id="moreHomes"></div>
-    <div id="app"></div>
+    <div id="topimages"></div>
+    <div id="descriptions"></div>
+    <div id="reviews"></div>
+    <div id="morehomes"></div>
 
     <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
     <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
 
-    <script src="http://localhost:3004/bundle.js"></script>
-    <script src="http://localhost:3000/bundle.js"></script>
+    <script src="https://s3-us-west-1.amazonaws.com/fecbundlefiles/top+images/bundle.js"></script>
```

Comment:
What is the point in hosting the bundles? Is this just so you don't have to manually start the servers?

Reply:
Do you need all the services as submodules in your repo if all you're using from them is their bundle file?

Reply:
I'm hosting them according to the description on Learn under project_milestones, but without the server files for each service, I'm having an issue with the GET requests not being sent to the specified routes on the specified ports.